### ISSUE TITLE
DI search crashing

### ISSUE THREAD

**Issue by PMSeitzer** [eugenemel/maven#269]
Only observed with whole `20200706-DIMS-negative` database vs `EDTAplasma_Sol02_25x_NegMeth09_37.mzML`, not observed for IS database

parameters:
```
lib_search_params <- list(
  "consensusMinFractionMs2Scans"=0.25,
  "ms2MinNumMatches"=2,
  "spectralCompositionAlgorithm"="AUTO_SUMMARIZED_ACYL_CHAINS_SUM_COMPOSITION",
  "ms1IsFindPrecursorIon"=TRUE,
  "ms1PpmTolr"=5,
  "ms1ScanFilter"="SIM"
)
```

**Comment by PMSeitzer** [comment]
Fixed as of https://github.com/eugenemel/maven_core/commit/fbb2e3d42fe4269be97a1ee9be9751c7da7005cf

**Comment by PMSeitzer** [comment]
This bug was introduced and fixed as a part of #267
